scripts/generators/tsconfig.js

```diff
@@ -2,6 +2,10 @@ import path from "path";
 import fs from "fs";
 import { createRequire } from "module";
 import { fileURLToPath } from "url";
+import globby from "globby";
+import * as babel from "@babel/core";
+import traverseModule from "@babel/traverse";
```

Comment:
when running it locally instead of default export - whole module is imported here

Reply:
It's expected, because `@babel/traverse` is not a native module but it's compiled to CJS.